packages/eddsa-poseidon/tests/index.test.ts

```diff
@@ -117,7 +117,7 @@ describe("EdDSAPoseidon", () => {
         expect(signature.S).toBe(circomlibSignature.S)
     })
 
-    it("Should sign a message (string)", async () => {
+    it("Should sign a message if less than 32 bytes (string)", async () => {
```

Comment:
I think this case shouldn't be supported without an explicit conversion to bigint